dotcms-integration/src/test/java/com/dotmarketing/portlets/templates/business/TemplateAPITest.java

```diff
@@ -51,20 +44,22 @@
 import com.dotmarketing.portlets.languagesmanager.model.Language;
 import com.dotmarketing.portlets.structure.model.Structure;
 import com.dotmarketing.portlets.templates.business.TemplateFactory.HTMLPageVersion;
-import com.dotmarketing.portlets.templates.design.bean.ContainerUUID;
+import com.dotmarketing.portlets.templates.design.bean.*;
 import com.dotmarketing.portlets.templates.model.FileAssetTemplate;
 import com.dotmarketing.portlets.templates.model.Template;
-import com.dotmarketing.util.Constants;
-import com.dotmarketing.util.Logger;
-import com.dotmarketing.util.UUIDGenerator;
-import com.dotmarketing.util.UtilMethods;
+import com.dotmarketing.util.*;
 import com.liferay.portal.model.User;
+
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
+
+import graphql.AssertException;
```

Comment:
why graphql?

Reply:
done https://github.com/dotCMS/core/pull/28128/commits/5009888ac50550c0d87e2f4b5fbce1fdd7ba6c08#diff-5dcfb4ccde2b8bfbbbe18e3a748ae7595e6007af7e248f5c48ee85a808d92f81L39

I was using the wrong class